<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CInfoBarStatusComponent</class>
 <widget class="QFrame" name="CInfoBarStatusComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>25</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>25</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Info status bar</string>
  </property>
  <layout class="QHBoxLayout" name="hl_InfoBarStatus">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="swift::gui::components::CTransponderModeComponent" name="comp_XpdrMode">
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbl_Network">
     <property name="toolTip">
      <string>Network</string>
     </property>
     <property name="text">
      <string>net</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignLeft">
    <widget class="swift::gui::CLedWidget" name="led_Network" native="true"/>
   </item>
   <item>
    <widget class="QLabel" name="lbl_Simulator">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Simulator</string>
     </property>
     <property name="text">
      <string>sim</string>
     </property>
     <property name="indent">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignLeft">
    <widget class="swift::gui::CLedWidget" name="led_Simulator" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbl_MapperReady">
     <property name="toolTip">
      <string>Mappings ready</string>
     </property>
     <property name="text">
      <string>map</string>
     </property>
     <property name="indent">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignLeft">
    <widget class="swift::gui::CLedWidget" name="led_MapperReady" native="true"/>
   </item>
   <item>
    <widget class="QLabel" name="lbl_DBus">
     <property name="text">
      <string>DBus</string>
     </property>
     <property name="indent">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignLeft">
    <widget class="swift::gui::CLedWidget" name="led_DBus" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbl_Ptt">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Push to talk</string>
     </property>
     <property name="text">
      <string>ptt</string>
     </property>
     <property name="indent">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignLeft">
    <widget class="swift::gui::CLedWidget" name="led_Ptt" native="true"/>
   </item>
   <item>
    <widget class="QLabel" name="lbl_Audio">
     <property name="toolTip">
      <string>Audio</string>
     </property>
     <property name="text">
      <string>au</string>
     </property>
     <property name="indent">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignLeft">
    <widget class="swift::gui::CLedWidget" name="led_Audio" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swift::gui::CLedWidget</class>
   <extends>QWidget</extends>
   <header>gui/led.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CTransponderModeComponent</class>
   <extends>QFrame</extends>
   <header>gui/components/transpondermodecomponent.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
